product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters.mdx

```diff
@@ -6,15 +6,86 @@ The database parameters listed on the DB Configuration tab are also referred to
+
+-  For more information on the types of values parameters accept, see [Parameter Names and Values](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). The types apply to both PostgreSQL and EDB Postgres Advanced Server parameters.
+
 
 !!!note
 Not all database configuration parameters are supported by BigAnimal. Some parameters, such as `wal_level` and `restore_command`, are reserved for EDB to provide the managed database features of BigAnimal. 
 !!!
 
+## Using formulas for parameter values
+
+In addition to entering specific values for parameters, for some parameters you can specify formulas to automatically calculate a value. You can use formulas for parameters of type integer and real using the following operators: `+ - / * > >= < <= == != && || ! ? : ( )`. Use `?` amd `:` in ternary formulas such as the [shared buffer example](#example). Use `( )` to specify [order of operations](order-of-operations), if needed. GUCs used in formulas must also be of type integer or real. All arithmetic is done on 64-bit floating point values rounded to an integer result.  
+
+BigAnimal has what we refer to as psuedo GUCs to facilitate creating equations. These read-only GUCs are:
+- `cpu_cores` &#8212 number of cores provided by the VM running the cluster nodes
+- `disk_iops` &#8212 number of IOPS allocated to the disks backing the primary data volume
+- `disk_size` &#8212 amount of memory backing the primary data volume, in KB
+- `ram` &#8212 amount of memory provided by the VM running the cluster nodes, in KB
+- `replica_count` &#8212 total number of nodes in the cluster
+
+Formulas can calculate a value relative to the specifications you selected for the cluster or relative to other non-pseudo GUCs. If you resize your cluster, BigAnimal recalculates formulas using the new values for the pseudo GUCs. 
+
+### Order of operations
+`*` and `/` are at the same operator precedence. `+` and `-` are at the same operator precedence. `*` and `/` are evaluated before `+` and `-`. Use parentheses to specify additional order of operations. 
+
+### Units
+GUCs can either be a bare number, such as 8, or have a unit, such as GB. Units are categorized into classes such as size or duration.
+
+You can't use GUCs with different classes of units in formulas. For example, you can't use a formula that tries to multiply two seconds with three kilobytes (2s \* 3KB), but you can use a formula that multiplies two sizes even if they have different scales (2KB \* 8GB). 
+
+The unit of the target GUC must match the unit used in the formula. For example, you can multiply two seconds by four days but you can't assign the result to a GUC of unit KB.
```

Comment:
Let's make the first sentence. "The kind of unit of the target GUC must match the kind of unit used in the formula." This is so that the user will know it's OK to use kB and MB in the same equation (different units but the same kind) whereas s and kB will not be allowed (because they are different kinds of units.

Reply:
@edbmanniwood I'm struggling with this one a little. The sentence before we define what we mean by classes of units. Can we use "class" instead of "kind"? or maybe "type" or "category"? Or do we need to change the preceding paragraph?

I left the proceeding paragraph and used "category" instead of "kind". Let me know what you think.

Reply:
My type theory brain suggests that "kind" is a better word to use in both places here than "category".